frontend/src/hooks/useMediaCapture.js

```diff
@@ -1,13 +1,15 @@
 import { useCallback } from 'react';
+import moment from 'moment';
 import html2canvas from 'html2canvas';
 
-export default function useMediaCapture(reference, title) {
+export default function useMediaCapture(reference, title = 'download') {
```

Comment:
Coming from Jon's coverage PR to this do we test the default value in a test (branch coverage).